core/src/test/java/com/mygdx/game/models/BrainstormingPhaseTest.java

```diff
@@ -0,0 +1,78 @@
+package com.mygdx.game.models;
+
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+public class BrainstormingPhaseTest {
+    int BRAIN_DAMAGE = 10;
+    int maxHitPoints = 100;
+
+    BrainstormingPhase bp;
+    Player p1;
+    Brain b1;
+    Brain b2;
+    String i1;
+    String i2;
```

Comment:
I can change them. The reason I used these variable names is because it was what got used in the test that were allready written. But I agree with you will try and change them to something more readable

Reply:
Yeah understandable, I am a bit slow when it comes to reviewing so I wasn't able to address these comments until now

Reply:
Thats totally okay ;)